share/completions/port.fish

```diff
@@ -0,0 +1,115 @@
+complete -f -n '__fish_port_no_subcommand' -c port -a 'mirror'  --description 'Create/update a local mirror of distfiles used for ports given on the command line'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'mdmg'  --description 'Creates an internet-enabled disk image containing an OS X metapackage of portname and its dependencies'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'mpkg'  --description 'Create binary archives of a port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'notes'  --description 'Displays notes for portname'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'outdated'  --description 'List outdated ports'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'patch'  --description 'Run patch phase of a port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'pkg'  --description 'Create binary archives of a port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'provides'  --description 'Find the port that installed a file'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'rdependents'  --description 'Recursively lists the installed ports that depend on the port portname'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'rdeps'  --description 'Recursively lists the other ports that are required to build and run portname'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'reload'  --description 'A shortcut to launchctl, like load and unload, but reloads the daemon'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'rev-upgrade'  --description 'Manually check for broken binaries and rebuild ports containing broken binaries'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'search'  --description 'Search for a port using keywords'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'select'  --description 'For a given group, selects a version to be the default by creating appropriate symbolic links'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'selfupdate'  --description 'Upgrade MacPorts itself and update the port definition files.'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'setrequested'  --description 'Mark portname as requested'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'setunrequested'  --description 'Mark portname as unrequested'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'sync'  --description 'Update the port definition files'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'test'  --description 'Run test phase of a port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'unarchive'  --description 'Extract the destroot of the given ports from a binary archive'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'uninstall'  --description 'Remove a previously installed port'
+complete -f -n '__fish_port_no_subcommand' -c port -a 'unload'  --description 'A shortcut to launchctl, like load, but unloads the daemon'
```

Comment:
This can just be "unload the daemon" - a similar change can be made to the `load` description.